io.elementary.Sdk.json

```diff
@@ -41,6 +41,32 @@
         }
     },
     "modules": [
+         {
+            "name": "gtk",
```

Comment:
Not sure exactly what's going on but when trying to build locally it fails like this:

```
Running custom install script '/usr/bin/python3 /run/build-runtime/gtk/build-aux/meson/post-install.py 3.0 3.0.0 /usr/bin /usr/lib/x86_64-linux-gnu /usr/share'
--- stdout ---
Compiling GSettings schemas...
Updating icon cache...
Updating module cache for input methods...

--- stderr ---
Warning: Schema “org.gnome.crypto.pgp” has path “/desktop/gnome/crypto/pgp/”.  Paths starting with “/apps/”, “/desktop/” or “/system/” are deprecated.
Warning: Schema “org.gnome.system.locale” has path “/system/locale/”.  Paths starting with “/apps/”, “/desktop/” or “/system/” are deprecated.
Warning: Schema “org.gnome.system.proxy” has path “/system/proxy/”.  Paths starting with “/apps/”, “/desktop/” or “/system/” are deprecated.
Warning: Schema “org.gnome.system.proxy.http” has path “/system/proxy/http/”.  Paths starting with “/apps/”, “/desktop/” or “/system/” are deprecated.
Warning: Schema “org.gnome.system.proxy.https” has path “/system/proxy/https/”.  Paths starting with “/apps/”, “/desktop/” or “/system/” are deprecated.
Warning: Schema “org.gnome.system.proxy.ftp” has path “/system/proxy/ftp/”.  Paths starting with “/apps/”, “/desktop/” or “/system/” are deprecated.
Warning: Schema “org.gnome.system.proxy.socks” has path “/system/proxy/socks/”.  Paths starting with “/apps/”, “/desktop/” or “/system/” are deprecated.
Traceback (most recent call last):
  File "/run/build-runtime/gtk/build-aux/meson/post-install.py", line 45, in <module>
    immodule_cache_file = open(os.path.join(gtk_moduledir, 'immodules.cache'), 'w')
OSError: [Errno 30] Read-only file system: '/usr/lib/x86_64-linux-gnu/gtk-3.0/3.0.0/immodules.cache'

FAILED: install script '/usr/bin/python3 /run/build-runtime/gtk/build-aux/meson/post-install.py 3.0 3.0.0 /usr/bin /usr/lib/x86_64-linux-gnu /usr/share' exit code 1, stopped
FAILED: meson-install 
/usr/bin/meson install --no-rebuild
ninja: build stopped: subcommand failed.
Error: module gtk: Child process exited with code 1
```